packages/stepper/src/types.ts

```diff
@@ -0,0 +1,63 @@
+import React, { Dispatch } from 'react';
+
+export interface StepperContextValues {
+  isDarkMode: boolean;
+  setIsDarkMode: Dispatch<React.SetStateAction<boolean>>;
+  stepIconClassName: string;
+  stepLabelClassName: string;
+}
+
+// TODO: replace with common type across all components
+export const Mode = {
```

Comment:
Don't forget to export `type Mode = typeof Mode[keyof typeof Mode];` 🙃

Reply:
Why do we need this? Not having `type Mode = typeof Mode[keyof typeof Mode];` doesn't seem to interfere with how we write dark mode styles.

Unless the syntax for dark mode styles I wrote in the `StepIcon` component isn't how we want to be writing dark mode styles - lmk if that's the case.

Reply:
It allows you to use `Mode` as both a constant and a type. We may not need it in this component, but it's best practice